kore/src/Kore/Attribute/Pattern/Simplified.hs

```diff
@@ -13,6 +13,11 @@ module Kore.Attribute.Pattern.Simplified
     , simplifiedConditionally
     , simplifiableConditionally
     , unparseTag
+    -- * For testing
```

Comment:
I don't think any of these exports are things that need to be hidden, so I don't think we need to express reservations about exporting them:
```suggestion
```